frontend/src/app.tsx

```diff
@@ -12,37 +13,47 @@ import Notifications from 'components/notifications/notifications';
 import Map from 'components/map/map';
 import Permissions from 'components/permissions/permissions-page';
 import { AuthorizedRoute } from 'components/common';
+import { AuthActionCreator } from 'store/slices';
 
-const App: React.FC = () => (
-  <Switch>
-    <Route path={[AppRoute.SIGN_IN, AppRoute.SIGN_UP]} component={Sign} />
-    <AuthorizedRoute path={AppRoute.NOTIFICATIONS} component={Notifications} />
-    <AuthorizedRoute
-      path={AppRoute.USER_PROFILE_$ID}
-      exact
-      component={Profile}
-    />
-    <AuthorizedRoute path={AppRoute.CLINICS} component={Clinics} />
-    <AuthorizedRoute path={AppRoute.DOCTORS} component={Doctors} />
-    <AuthorizedRoute
-      path={AppRoute.DOCTOR_DETAILS_$ID}
-      component={DoctorDetails}
-    />
-    <AuthorizedRoute
-      exact
-      path={AppRoute.USERS}
-      component={Users}
-      permissions={[PermissionName.CREATE_USER, PermissionName.EDIT_USER]}
-    />
-    <AuthorizedRoute
-      exact
-      path={AppRoute.MAP}
-      component={Map}
-      permissions={[PermissionName.MAP_MANIPULATION]}
-    />
-    <AuthorizedRoute path={AppRoute.PERMISSIONS} component={Permissions} permissions={[PermissionName.EDIT_PERMISSIONS]} />
-    <Route path="*" exact component={NotFound} />
-  </Switch>
-);
+const App: React.FC = () => {
+  const user = JSON.parse(localStorage.getItem('user') as string);
```

Comment:
We don't have a user in the localstorage and we don't want to store the user there